.github/workflows/plugins-ci.yml

```diff
@@ -122,7 +303,12 @@ jobs:
     if: >
         github.event_name == 'pull_request' &&
         github.event.pull_request.user.login == 'dependabot[bot]'
-    needs: test
+    needs:
+      - test
+      - test-postgres
+      - test-mongo
+      - test-mysql
+      - test-redis
```

Comment:
Not 100% this will work @Uzlopak, needing conditionally run jobs causes the job needing them to be skipped.

See https://github.com/fastify/workflows/pull/46

Reply:
now i use https://github.com/actions/runner/issues/491#issuecomment-850884422

better?